ccp/index_reader.py

```diff
@@ -265,11 +265,15 @@ class BuildConfigManager(object):
     by pipeline service
     """
 
-    def __init__(self, registry_url, namespace, from_address, smtp_server):
+    def __init__(self, registry_url, namespace, from_address, smtp_server,
+                 ccp_openshift_slave_image, notify_cc_emails):
```

Comment:
parameters which needed to be passed from build template to template

Reply:
Guess - I'll use the jenkins REST APIs to fetch these details in IndexReader later as we did in notifications module. ;-)

Dont wanted to overload this PR.